docs/http.md

```diff
@@ -0,0 +1,61 @@
+# HTTP API
```

Comment:
Why not inline this in the README for now for maximum discoverability? I think fat READMEs are underrated, and I think this is high level enough to be there. I always Cmd-F for stuff

Reply:
Hmm, yes. [I like long web pages too.](https://clig.dev/) We're thinking of turning this into some proper docs at some point too, so we'll have to think in context of that.

The downside of one long page is then search engines suffer. You can't google "Cog HTTP API" and get deep linked to the right place.